nginx/nginx.conf

```diff
@@ -44,10 +44,12 @@ http {
         ssl_certificate             /etc/nginx/private/cert.pem;
         ssl_certificate_key         /etc/nginx/private/key.pem;
         ssl_prefer_server_ciphers   on;
-        ssl_session_timeout         30m;
-        ssl_session_cache           shared:SSL:5m;
-        ssl_protocols               TLSv1.2 TLSv1.1 TLSv1;
-        ssl_ciphers                 ECDHE-RSA-AES256-SHA:DHE-RSA-AES256-SHA:DHE-RSA-CAMELLIA256-SHA:DHE-RSA-AES128-SHA:DES-CBC3-SHA:!EXP:!ADH:!aNULL;
+        ssl_session_timeout         10m;
+        ssl_session_cache           shared:SSL:10m;
+        ssl_protocols               TLSv1.2 TLSv1.3;
```

Comment:
yes! so great to see `v1` and `v1.1` get dropped!  Welcome `v1.3`, lookin' good ; )